benchmarks/tpcc/src/main/java/com/google/cloud/pgadapter/tpcc/entities/District.java

```diff
@@ -0,0 +1,137 @@
+package com.google.cloud.pgadapter.tpcc.entities;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.EmbeddedId;
+import jakarta.persistence.Entity;
+import jakarta.persistence.FetchType;
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.ManyToOne;
+import jakarta.persistence.Table;
+import java.math.BigDecimal;
+
+@Entity
+@Table(name = "district")
+public class District {
+
+  @EmbeddedId private DistrictId id;
+
+  @ManyToOne(fetch = FetchType.LAZY)
```

Comment:
Same here also as for Customer: This should probably be EAGER, if we in most cases also call get warehouse() after having fetched a District.